src/CTA.WebForms2Blazor/ControlConverters/ButtonControlConverter.cs

```diff
@@ -9,19 +10,29 @@ public class ButtonControlConverter : ControlConverter
         protected override Dictionary<string, string> AttributeMap { 
             get
             {
-                throw new NotImplementedException();
+                return new Dictionary<string, string>()
+                {
+                    ["onclick"] = "@onclick", 
+                    ["cssclass"] = "class"
+                };
             } 
         }
-        protected override string BlazorName { get { throw new NotImplementedException(); } }
-        
-        public ButtonControlConverter() : base()
-        {
-            
-        }
+        protected override string BlazorName { get { return "button"; } }
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            string buttonText = "";
+            try
+            {
+                var textAttr = node.Attributes.AttributesWithName("text").Single();
+                buttonText = textAttr.Value;
+            }
+            catch (InvalidOperationException ex)
+            {
+                // Todo: Throw warning about button with no text
+            }
```

Comment:
Not sure if you saw this comment from earlier, but do you think adding a default value would work? Example:

```csharp
var textAttr = node.Attributes.AttributesWithName("text").FirstOrDefault();
var buttonText = textAttr?.Value ?? string.Empty;
```
This sets a default value of string.Empty if the text attribute is null/absent.

Reply:
I think thats what I have right now, the textAttr is initially set to be an empty string and if there is a text attribute then it gets updated with that value, otherwise it stays an empty string

Reply:
Good point, you’re right, it is what you have. I do think we should adjust the logic to avoid the try/catch though, because the edge case being handled is valid and exceptions should be reserved for when we encounter invalid state. The exception we're handling is caused by our implementation, which is within our control.